modules/light-clients/08-wasm/internal/types/store.go

```diff
@@ -48,7 +48,7 @@ func NewMergedClientStore(subjectStore, substituteStore storetypes.KVStore) Merg
 // Get implements the storetypes.KVStore interface. It allows reads from both the subjectStore and substituteStore.
 //
 // Get will return an empty byte slice if the key is not prefixed with either "subject/" or "substitute/".
-func (ws MergedClientStore) Get(key []byte) []byte {
+func (ws ClientRecoveryStore) Get(key []byte) []byte {
```

Comment:
may as well refactor all receiver args to `s`?

 ```suggestion
func (s ClientRecoveryStore) Get(key []byte) []byte {
```